src/main/resources/db/migration/changelog-master.xml

```diff
@@ -10,6 +10,7 @@
 	<include file="changelog-v0.0.3.xml" 	relativeToChangelogFile="true"/>
 	<include file="changelog-v0.0.4.xml" 	relativeToChangelogFile="true"/>
 	<include file="changelog-v0.0.5.xml" 	relativeToChangelogFile="true"/>
+	<include file="changelog-v0.0.7.xml" 	relativeToChangelogFile="true"/>
```

Comment:
Pull and merge the latest develop branch; there's a conflict here, but doing that should resolve it.